client/src/theme/styles/components/frontend/project-collection/_header.scss

```diff
@@ -0,0 +1,141 @@
+$_icon-size: 60px;
+$_hero-max-height: 340px;
+$_gap-size: 30px;
+
+.project-collection-header {
+  @include templateHead;
+  display: grid;
+  grid-template-areas:
+    'title'
+    'description';
+  grid-gap: $_gap-size;
+  margin-bottom: $_gap-size;
+  color: $neutral90;
+
+  // For the configurations where the description is at the
+  // bottom of the grid, but there is no description present.
+  // The cell will still exist but the content will not, so the
+  // grid-gap will serve as an unwanted additional bottom padding.
+  &--no-description {
+    margin-bottom: 0;
+  }
+
+  &--square {
+    grid-template-areas:
+      'title'
+      'image'
+      'description';
+
+    @include respond($break75) {
+      /* stylelint-disable declaration-block-no-redundant-longhand-properties */
+      // There is a bug where using 1fr as a unit in grid-template will cause an
+      // SCSS compilation error
+
+      grid-template-areas:
+        'image title'
+        'image description';
+      grid-template-rows: auto 1fr;
+      grid-template-columns: $_hero-max-height auto;
+      /* stylelint-enable declaration-block-no-redundant-longhand-properties */
+
+      // Add the margin bottom back in for the square configuration
+      // where the description is not at the bottom of the grid.
+      margin-bottom: $_gap-size;
+    }
+  }
+
+  &--wide {
+    grid-template-areas:
+      'title'
+      'image'
+      'description';
+  }
+
+  &--full {
+    grid-template-areas:
+      'image'
+      'title'
+      'description';
+  }
+
+  &__header {
+    display: flex;
+    grid-area: title;
+    align-items: center;
+    transition: color $duration $timing;
+
+    &--link {
+      text-decoration: none;
+
+      &:visited {
+        color: $neutral90;
+      }
+
+      &:hover {
+        @include defaultHoverStyle;
```

Comment:
@ConnorChristensen it looks like this declaration is redundant with the default hover color of `a` tags. As I mentioned, try to use the defaults whenever possible to keep the code DRY.

Reply:
Good catch. Thank you.